pkg/testing/v1alpha1/revision.go

```diff
@@ -96,63 +96,64 @@ func WithRevStatus(st v1alpha1.RevisionStatus) RevisionOption {
 
 // MarkActive calls .Status.MarkActive on the Revision.
 func MarkActive(r *v1alpha1.Revision) {
-	r.Status.MarkActive()
+	r.Status.MarkActiveTrue()
 }
 
 // MarkInactive calls .Status.MarkInactive on the Revision.
 func MarkInactive(reason, message string) RevisionOption {
 	return func(r *v1alpha1.Revision) {
-		r.Status.MarkInactive(reason, message)
+		r.Status.MarkActiveFalse(reason, message)
 	}
 }
 
 // MarkActivating calls .Status.MarkActivating on the Revision.
 func MarkActivating(reason, message string) RevisionOption {
 	return func(r *v1alpha1.Revision) {
-		r.Status.MarkActivating(reason, message)
+		r.Status.MarkActiveUnknown(reason, message)
 	}
 }
 
 // MarkDeploying calls .Status.MarkDeploying on the Revision.
 func MarkDeploying(reason string) RevisionOption {
 	return func(r *v1alpha1.Revision) {
-		r.Status.MarkDeploying(reason)
+		r.Status.MarkResourcesAvailableUnknown(reason, "")
```

Comment:
Is there a reason we are removing some of the `MarkXxx` methods off of revision status?

Reply:
`MarkDeploying()` is not a `ConditionType` but a helper function that alters two condition types. No longer needed if we explicitly set condition types.

https://github.com/knative/serving/blob/1d263950f9f2fea85a4dd394948a029c328af9d9/pkg/apis/serving/v1alpha1/revision_lifecycle.go#L171-L174